src/graphql/resolvers/index.js

```diff
@@ -254,15 +273,18 @@ module.exports = function Resolvers (ssb) {
     })
   }
 
+
   return {
     Query: {
-      getMyRoom () {
+      getMyRoom (_, opts) {
```

Comment:
this seems a bit redundent.... you pass and opts.language in then if gives it back to you....
maybe just return the DEFAULT_LANGUAGE_CODE .... if it's even needed?
if we're not using this just rip it out

Reply:
oh i see it's used in the description... got it